src/boot/efi/secure-boot.c

```diff
@@ -11,3 +11,13 @@ BOOLEAN secure_boot_enabled(void) {
 
         return !EFI_ERROR(err) && secure;
 }
+
+#ifdef SBAT_DISTRO
+static const char sbat[]
+__attribute__((section (".sbat")))
+__attribute__((__used__))
+__attribute__((aligned(512))) = ""
```

Comment:
so, in the Linux userpace codebase we have three macros for these gcc attributes. We migrated some of the other macros for gcc attributes into a generic header src/fundamental/macro-fundamental.h, that is shared by the EFI and Linux userspace codebase.

I'd suggest we do the same for these three attributes. i.e. migrate the existing macros for them from src/basic/macro to src/fundamental/macro-fundamental.h, and then use them here.

Also, please use `__` prefix and postfix for all words in these attributes. We so far used them on all of them, so we should continue here. i.e. `section` → `__section__` and `aligned` → `__aligned__`.

Reply:
Everything else in macro.h & macro-fundationtal.h uses _foo_ style with single underscores, not double. So I carried on using single underscores. I hope that's what you meant.